hack/update-codegen.sh

```diff
@@ -9,7 +9,7 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
 
 verify="${VERIFY:-}"
 
-for group in apps authorization build config console image network oauth project quota route security template user; do
+for group in apps authorization build config console image imageregistry network oauth project quota route security template user; do
```

Comment:
how about adding `samples` in addition to `imageregistry` since you are in here @ricardomaraschini :-)

last time @adambkaplan and I talked, we discussed adding both new apis at the same time

Reply:
sure thing.

Reply:
done.

Reply:
thanks man :-)